charts/spire/charts/spire-server/values.yaml

```diff
@@ -187,10 +187,10 @@ controllerManager:
     failurePolicy: Fail
     upgradeHook:
       image:
-        registry: cgr.dev
-        repository: chainguard/kubectl
+        registry: docker.io
+        repository: rancher/kubectl
         pullPolicy: IfNotPresent
-        version: latest
+        version: ""
```

Comment:
Maybe a comment here that it will auto detect the version if its not specified here?